src/averageAges.js

```diff
@@ -36,8 +50,31 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
+function agesSumPerson(avaregeAgesPeople) {
```

Comment:
It's better to name something like getTotalPeopleAge in this case or something similar